spectroscopy/spectra_generator.md

```diff
@@ -42,7 +42,7 @@ The notebook may focus on the COSMOS field for now, which has a large overlap of
 | ------- | ------- | ----------- | ------------ | ------ |
 | IRSA    | Keck    | About 10,000 spectra on the COSMOS field from [Hasinger et al. (2018)](https://ui.adsabs.harvard.edu/abs/2018ApJ...858...77H/abstract) | [IRSA Archive](https://irsa.ipac.caltech.edu/cgi-bin/Gator/nph-scan?projshort=COSMOS) | Implemented with `astroquery.ipac.irsa`. (Table gives URLs to spectrum FITS files.) Note: only implemented for absolute calibrated spectra. |
 | IRSA    | Spitzer IRS | ~17,000 merged low-resolution IRS spectra | [IRS Enhanced Product](https://irsa.ipac.caltech.edu/cgi-bin/Gator/nph-dd?catalog=irs_enhv211) | Implemented with `astroquery.ipac.irsa`. (Table gives URLs to spectrum IPAC tables.) |
-| IRSA    | IRTF*        | Large library of stellar spectra | | does `astroquery.ipac.irsa` work?? |
+| ~~IRSA~~    | ~~IRTF~~        | ~~Large library of stellar spectra~~ | | ~~Not included here because the data are not currently available in an easily accessible format~~|
```

Comment:
we might want to add a hidden cell that lists all the spectra that we considered to include but rejected for various reasons?

Reply:
done.